schoolopy/main.py

```diff
@@ -1845,7 +1930,7 @@ def create_grading_categories(self, categories, section_id):
         :param categories: List of GradingCategory objects to create.
         :param section_id: ID of section in which to create categories.
         """
-        return [GradingCategory(raw) for raw in self._post('sections/%s/grading_categories' % section_id, {'grading_categories': {'grading_category': [category.json() for category in categories]}})['grading_category']]
+        return [GradingCategory(raw) for raw in self._post('sections/%s/grading_categories' % section_id, {'grading_categories': {'grading_category': [category for category in categories]}})['grading_category']]
```

Comment:
This is an example of where removing `.json()` would be a bad idea. I think I gave you bad advice.